config/rbac/role.yaml

```diff
@@ -938,6 +944,12 @@ rules:
   - deletecollection
   - get
   - patch
+- apiGroups:
+  - networking.istio.io
+  resources:
+  - gateways
+  verbs:
+  - '*'
```

Comment:
is this a good approach? shouldn;t we limit it to just what we really need to do? get/list for reading resources, etc.

Reply:
IDK, I'll let @bartoszmajsak comment here.

I can say that we do need edit/create privileges.

Reply:
We should narrow it down, I will take care of it.

Reply:
I created a separate issue for that, as this was set as "*" from the get-go. #732